app/views/events/participants.html.erb

```diff
@@ -7,44 +7,94 @@
-          <% end %>
-        </td>
-        <% if can? :print_agreement_letters, Event and @has_agreement_letters %>
-          <td>
-            <%= check_box_tag('selected_participants[]', participant.id, false, class: 'participant_selection') %>
-          </td>
-        <% end %>
+  <table class="table table-striped">
+    <tr>
+      <th><%= t 'activerecord.attributes.profile.name' %></th>
+      <th><%= t 'activerecord.attributes.profile.email' %></th>
+  	  <th><%= t 'agreement_letters.agreement_letter' %></th>
+
+      <% if can? :print_agreement_letters, Event and @has_agreement_letters %>
+      <th>
+        <label>
+          <input type="checkbox" id="select_all_participants" class="participant_selection" onclick="flipAllCheckboxes(this)">
+          <%= t 'events.agreement_letters_download.select_all' %>
+        </label>
+      </th>
+     <% end %>
+     <% if can? :show_eatings_habits, Event %>
+      <th>
+        <%= sort_caret(t('activerecord.methods.application_letter.eating_habits'), 'eating-habits') %>
+      </th>
+      <% end %>    
+    </tr>  
+  
+    <%  sort_participants_application_letters 
+    @participants.each do |participant| 
+       application_letter = ApplicationLetter.where(user_id: participant.id, event_id: @event.id).first
+
+
+
+      %>
+    <tr>
+      
+      <td>
+      <%= participant.name %>
+
+      </td>
+      <td>
+      <%= participant.email %>
+      </td>
+     
+      <td>
+      <% if participant.requires_agreement_letter_for_event?(@event) %>
+        <span class="text-muted"> <%= t '.unnecessary' %> </span>
+      <% elsif participant.agreement_letter_for_event?(@event) %>
+        <span class="text-success" title="<%= participant.agreement_letter_for_event(@event).updated_at %>"> <%= t '.available' %> </span>
+      <% else %>
+        <span class="text-danger"><%= t '.unavailable' %></span>
+      <% end %>
+    </td>
+    <% if can? :print_agreement_letters, Event and @has_agreement_letters %>
+      <td>
+        <%= check_box_tag('selected_participants[]', participant.id, false, class: 'participant_selection') %>
+      </td>
+    <% end %>
+      <% if can? :show_eatings_habits, Event %>
+      <td>
+         <% if application_letter.vegan || application_letter.vegeterian || application_letter.allergic %>
+           <ul id="eating-habit--list">
+           <% if application_letter.vegan %>
+             <li>
+               <img 
+                 class="img img-responsive eating-habit--icon"
+                 src="<%= image_url('application_letter_vegan.png') %>"
+               >
+             </li>
+           <% end %>                
+           <% if application_letter.vegeterian %>
+             <li>
+               <img 
+                 class="img img-responsive eating-habit--icon"
+                 src="<%= image_url('application_letter_vegeterian.png') %>"
+               >
+             </li>
+           <% end %>
```

Comment:
Yes it is, but the logic of this model as well as the well as the validation of user input is not on the side of my user story. I can follow your argument but that is to much interpretation adn it would be part of another user story.